### ISSUE TITLE
Check consistency between optics and sequence

### ISSUE THREAD

**Issue by sterbini** [lhcopt/lhcmask#6]
Hi,

one could consider to add in the sequence and in the optics the variables
- seq_version
- opt_version,
respectively.
Starting from that one could check/assert the consistency between the files.
G.

**Comment by freddieknets** [comment]
Good idea; this could then replace my ver_lhc_run and ver_hllhc_optics variables.
However, how would you number the difference between LHC and HL-LHC?
MAD-X does not take strings
